src/devices/heiman.ts

```diff
@@ -47,10 +47,38 @@ const fzLocal = {
                 result.occupancy = bit0 === 1;
                 result.sensor_status = ["none", "activity"][bit1to3] || "unknown";
                 result.fall_status = ["normal", "fall_warning", "fall_alarm"][bit4to5] || "unknown";
+            } else if (Object.hasOwn(msg.data, "ultrasonicOToUDelay")) {
+                result.radar_delay_time = msg.data.ultrasonicOToUDelay;
             }
+
             return result;
         },
     } satisfies Fz.Converter<"msOccupancySensing", undefined, ["attributeReport", "readResponse"]>,
+    diagnosticHeiman: {
```

Comment:
`m.numeric` can also be used here (https://github.com/Koenkk/zigbee-herdsman-converters/pull/10358/files#r2442292908)